guides/plugins/apps/tax-provider.md

```diff
@@ -0,0 +1,140 @@
+# Tax provider
+
+Tax calculations differ from country to country. Especially in the US, the sales tax calculation can be tedious, as the laws and regulation differ from state to state or even county or certain city taxes can apply. Therefore, most shops use a third-party service (so called tax-providers) to calculate sales taxes for them.
+
+Starting with version 6.5.0.0, Shopware allows apps to integrate custom tax calculation, which could include an automatic tax calculation with a tax provider. An app has to provide an endpoint, which is called during checkout and provide new tax rates. The requests and responses of all of your endpoints will be signed and feature JSON content.
+
+## Prerequisites
+
+You should be familiar with the concept of Apps and their registration.
```

Comment:
"concept of Apps" in other lines the term app is written in all lower case. Unsure if there is an intention to write it like that